babel.config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["next/babel"]
+}
```

Comment:
Can we rename this file to `.babelrc`? Both `.babelrc` and `babel.config.json` should work, but `.babelrc` is more common and some tooling that we might want to add in the future (such as Storybook) rely on the file being `.babelrc`